lib/id3.dart

```diff
@@ -236,7 +236,10 @@ class MP3Instance {
           apic['picType'] = PICTYPE[frame.readBytes(1).first] ?? 'Unknown';
           apic['description'] = frame.readString(checkEncoding: false);
           apic['base64'] = base64.encode(frame.readRemainingBytes());
-          metaTags['APIC'] = apic;
+          if (!metaTags.containsKey('APIC')) {
+            metaTags['APIC'] = [];
+          }
+          metaTags['APIC'].add(apic);
```

Comment:
should we add a new key "APIC_FRAMES" which will be an array that keep APIC as is, for backward compatibility?

cc: @moffatman